auth-web/src/components/auth/manage-business/CreateNewBusinessHelp.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div id="help-resolution" class="mb-12">
+    <div class="ma-6">
+      <div class="help-btn top" @click="helpToggle = !helpToggle">
+        <v-icon color="primary">mdi-help-circle-outline</v-icon>
+        <span v-if="!helpToggle" class="pl-2">{{ helpContent.toggleHelpShow }}</span>
+        <span v-else class="pl-2">{{ helpContent.toggleHelpHide }}</span>
+      </div>
+      <v-expand-transition v-if="helpToggle">
+        <section class="help-section info-text">
+          <header class="help-header mt-4">
+            <p class="title-text mt-4">{{ helpContent.title }}</p>
+          </header>
+          <br>
+          <div class="help-text mt-4">
+            <p>{{ helpContent.intro }}</p>
+            <ol>
+                <li class="mt-12" v-for="(step, index) in helpContent.steps" :key="index">
+                    <strong>{{ step.title }}</strong>
+                    <ul class="mt-8">
+                    <li v-for="(item, i) in step.items" :key="i">
+                        <p v-if="typeof item === 'string'">{{ item }}</p>
+                        <p v-else v-html="item.content"></p>
```

Comment:
Why not always use v-html?

Reply:
So you're still doing this and it's limiting how you display and style the steps and links/sublinks.

I recommend you just make this a static page -- HTML only, no v-if or anything. Then you can make it look exactly like the uxpin.